samples/bot-conversation-sso-quickstart/js/README.md

```diff
@@ -76,6 +76,8 @@ The sample uses the bot authentication capabilities in [Azure Bot Service](https
 
 - Update the `.env` configuration for the bot to use the Microsoft App Id and App Password from the step 1 (AAD app registration in Azure portal or from Bot Framework registration. (Note the MicrosoftAppId is the AppId created in step 1, the MicrosoftAppPassword is referred to as the "client secret" in step1 and you can always create a new client secret anytime.)
     Also, update `connectionName` as the name of your Azure Bot connection created in step 1.
+     - `MicrosoftAppType` - Set this as MultiTenant to if your bot is supported on multiple tenants; SingleTenant otherwise.
```

Comment:
This should be 
Set this as MultiTenant to if your bot is supported on multiple tenants; SingleTenant otherwise.

Like:
Set this as MultiTenant to if your bot is supported on multiple tenants, otherwise SingleTenant.